### ISSUE TITLE
Windows installer fails - can't download elixir.csv

### ISSUE THREAD

**Issue by frunox** [elixir-lang/elixir-windows-setup#22]
Tried to install a new version of Elixir today on Windows 10.  The installer crashes with the message:
"Error - downloading https://elixir-lang.org/elixir-csv failed.  Setup cannot continue."

I was able to download the .csv file, but the installer can't.  Is there a way to proceed?

**Comment by josevalim** [comment]
Does it really say elixir-csv instead of elixir.csv? If so, could that be the root cause? Otherwise it is something related with the https certificates in your machine.

**Comment by josevalim** [comment]
Oh, alternatively you can try using the previous version of the installer, but note that it uses a http address instead of https: https://github.com/elixir-lang/elixir-windows-setup/releases/download/v2.1/elixir-websetup.exe

**Comment by frunox** [comment]
That version has the same issue for me.

**Comment by josevalim** [comment]
Sorry to hear. Can you use chocolatey or similar? Otherwise follow the steps to install precompilled. Btw,. are you inside some network that may use proxies?

**Comment by frunox** [comment]
I am not on a network using proxies.
I am trying to install chocolatey but am getting errors in PowerShell trying to 'Set-ExecutionPolicy Bypass ....'.  This is after copying in the command from the Chocolatey website.
Where are the steps to install a precompiled version?

**Comment by frunox** [comment]
I got chocolatey to install - there was a problem connecting to the server initially.

I uninstalled my current versions of Erlang and Elixir, then installed via chocolately.  The install appeared to work.  However, nothing is working.  When I type 'elixir -v' from the terminal, I get this error:
```
{"init terminating in do_boot",{'cannot get bootfile','C:\\Program Files\\erl-24.1/bin/start.boot'}}
init terminating in do_boot ({cannot get bootfile,C:\Program Files\erl-24.1/bin/start.boot})
```

I had Erlang 24.1 installed before - chocolately installed 24.0.  Something is still wrong with my installation.  Where can I get help?
